tests/foreman/ui/test_hardwaremodel.py

```diff
@@ -55,9 +54,10 @@ def test_positive_end_to_end(session, module_org, module_location):
         assert hm_values['vendor_class'] == vendor_class
         assert hm_values['info'] == info
         # Create host with associated hardware model
-        host_name = create_fake_host(
-            session, host_template, extra_values={'additional_information.hardware_model': name}
-        )
+        host = session.host.create(values)
+        values.update({'additional_information.hardware_model': name})
+        host_name = f'{name}.{host.domain.name}'
+        session.host.update(host_name, {'additional_information.hardware_model': name})
```

Comment:
```suggestion
        values.update({'additional_information.hardware_model': name})
        host_name = f'{name}.{host.domain.name}'
        host = session.host.create(values)
```
??

Reply:
TBH, I really didnt like the approach of removing the create_fake_host method and merge its code in tests directly! I believe we made it complex whereas earlier it was easy to identify that we are creating a fake host by just calling that function !!

Especially we are defining the `populate_ui_host_fields` in one test module and calling in another is an insane change! 

I would have moved create_fake_host in satellite class mixins ! so that I could have access it as `sat.create_fake_host()` or in api_utils to access as `sat.api_utils.create_fake_host()`. This would have been easier outside automated tests as to quick deploy the host from python interactive shell !